build.sh

```diff
@@ -86,6 +86,43 @@ dnf config-manager --set-disabled copr:copr.fedorainfracloud.org:che:nerd-fonts
 dnf -y --enablerepo copr:copr.fedorainfracloud.org:che:nerd-fonts install \
   nerd-fonts
 
+# UUPD while it doesnt have a COPR
+UUPD_INSTALL=$(mktemp -d)
+curl --retry 3 -Lo $UUPD_INSTALL/uupd.tar.gz https://github.com/ublue-os/uupd/releases/download/v0.5/uupd_Linux_x86_64.tar.gz
+tar xzf $UUPD_INSTALL/uupd.tar.gz -C $UUPD_INSTALL
+cp $UUPD_INSTALL/uupd /usr/bin/uupd
+rm -rf $UUPD_INSTALL
+tee /usr/lib/systemd/system/uupd.service <<EOF
```

Comment:
Can we have the service units just in the repo in the right directories instead of inlining them in here? This file is going to get messy.

Reply:
I completely forgot we could do that?